ydl_server/ydlhandler.py

```diff
@@ -113,7 +113,7 @@ def fetch_metadata(url):
     stdout = io.StringIO()
     stderr = io.StringIO()
     info = None
-    with ydl_module.YoutubeDL({'extract_flat': 'in_playlist'}) as ydl:
+    with ydl_module.YoutubeDL({'extract_flat': 'in_playlist', 'cookiefile': app_config.get('ydl_options',{}).get('cookies','../cookies.txt')}) as ydl:
```

Comment:
It can not fetch_metadata without pass the cookies file here.
If it fails here, it won't go into download.

Reply:
Here can also be changed to judge that the file does not exist, then the cookiefile parameters don't passed in.
But I think it is better to prepare an empty cookie file. It can also be a reference for other users when they are adding.